### PR TITLE
My-account-page-Recently-Ordered-check-box-misaligned-on-tab-portrait…

### PR DESCRIPTION
**Pull request by ajay2jcommerce**
…-view ::my account page check box misaligned on tab portrait view

### Fixed Issues (if relevant)

1. My account page "Recently Ordered" check box misaligned on tab portrait view #20143
2. ...

### Manual testing scenarios (*)

1. placed order with login customer on my account page in sidebar check resend order product.

2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @ajay2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=20155) <br/>All committers have signed the CLA.

**Comment by ajay2jcommerce** [comment]
Hi @magento-engcom-team please review.

**Comment by ajay2jcommerce** [comment]
Hi @magento-engcom-team @miguelbalparda please review.

File: app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less [hunk 1]
```diff
@@ -343,16 +343,22 @@
         }
 
         .product-item-name {
-            display: inline-block;
+            float:left;
+            width:calc(100% - 20px);
```
Comment (zanilee) [inline app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less]: Please add a space after ":" according to Magento Less coding standard: https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-less.html#properties-colon-indents

**Comment by sdzhepa** [comment]
Hi @ajay2jcommerce 

Could you please provide style fixes according to code review comments?

Thank you for contribution and collaboration!

File: app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less (hunk 1 above)
Comment (ajay2jcommerce) [reply app/design/frontend/Magento/luma/Magento_Sales/web/css/source/_module.less]: Ok

**Comment by ajay2jcommerce** [comment]
Hi @zanilee, @sdzhepa  updated code as per style fixes according to code review comments.
Please review.

Thanks

**Comment by sdzhepa** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [Magento instance](https://i-20155-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-20155-2-3-develop.instances.magento-community.engineering/admin](https://i-20155-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by sdzhepa** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, here is your [new Magento instance](https://pr-20155.instances.magento-community.engineering).
Admin access: [https://pr-20155.instances.magento-community.engineering/admin](https://pr-20155.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by sdzhepa — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sdzhepa, thank you for the review.
[ENGCOM-4168](https://jira.corp.magento.com/browse/ENGCOM-4168) has been created to process this Pull Request

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [Magento instance](https://i-20155-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-20155-2-2-develop.instances.magento-community.engineering/admin](https://i-20155-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Pull request merged**